frontend/apps/marketing/src/components/nextImage/NextImage.tsx

```diff
@@ -0,0 +1,12 @@
+import BaseNextImage, {ImageProps} from 'next/image';
+
+const NextImage = (props: ImageProps) => {
+  const baseImageProps: ImageProps = {
+    ...props,
+    fill: props?.fill ? props.fill : true, // Fill by default to make responsive and avoid absolute width/height
+  };
+
+  return <BaseNextImage {...baseImageProps} />;
+};
```

Comment:
```suggestion
const NextImage: React.FC<ImageProps> = ({fill = true, ...props}) => (
  <BaseNextImage fill={fill} {...props} />
);

Reply:
how about this?  As it's more easier to read and is more consistent with our codebase